README.md

```diff
@@ -86,14 +86,14 @@ Options:
                                                source control since this tool will make
                                                large changes to both the project and
                                                source files.
-  -e, --extension <extension>                  Specifies a .NET Upgrade Assistant
+  --extension <extension>                      Specifies a .NET Upgrade Assistant
                                                extension package to include. This could
                                                be an ExtensionManifest.json file, a
                                                directory containing an
                                                ExtensionManifest.json file, or a zip
                                                archive containing an extension. This
                                                option can be specified multiple times.
-  --entry-point <entry-point>                  Provides the entry-point project to start
+  -e, --entry-point <entry-point>              Provides the entry-point project to start
                                                the upgrade process.
```

Comment:
Probably makes sense to add a line on globbing pattern that users can use? like ```upgrade-assistant <solution> -e *``` will run all projects in the solution.